src/Navigation/Notifications/Repository/Notify.php

```diff
@@ -796,7 +798,7 @@ public function createFromNotification(Entity\Notification $Notification): bool
 				}
 			}
 		}
-
+// todo - check
```

Comment:
This is incomplete, even for debugging purposes. Please expand on it or remove it.

Reply:
Done